### DISCUSSION TITLE
Build problems

### DISCUSSION THREAD

**Discussion by chuyanjuncn** [pelicanmapping/osgearth#2076] [Support Q&A] [answered]
Is it possible to provid osgearth with all the dependency source code and CMakelist.  Like what is in VTK,  there is one  CMakelist.txt, if you want to build it,  just use  the main CMakeList.txt，all the required dependency lib (or 3dparty lib) and VTK itself will be automaticaly built.

**Comment by jasonbeverage** [comment, marked as answer]
The closest you'll get to that is using vcpkg as described here  http://docs.osgearth.org/en/latest/build.html on windows.    On Linux you can install packages easily from your package manager.  If you look at the GitHub actions scripts in our repo you can get a feel for how to build osgearth on windows, Linux and osx from the command line with most of the dependencies.
